README.md

```diff
@@ -28,9 +28,9 @@ Run the following command lines to add apt-lines for APT repository on
 packages.red-data-tools.org:
 
 ```console
-$ sudo apt install -y -V ca-certificates lsb-release wget
```

Comment:
Why do you want to remove those options?
If we have them, users can install by copy & paste.

Reply:
This section will be reverted as it is expected that many users will copy and paste.

Reply:
By the way, if your priority is to be able to copy and paste, I think we should remove the `$` at the beginning of the line.